packaging/bundle-validation/README.md

```diff
@@ -50,4 +50,20 @@ Note that for each library like Hive and Spark, the download and extraction happ
 only one layer is generated to limit the size of the image. However, this makes repeated downloads when rebuilding the
 image. If you need faster iteration for local build, you may use the `Dockerfile` under `base-dev/`, which uses `ADD`
 instruction for downloads, which provides caching across builds. This increases the size of the generated image compared
-to `base/` and the image should only be used for development only and not be pushed to remote.
\ No newline at end of file
+to `base/` and the image should only be used for development only and not be pushed to remote.
+
+## Running Bundle Validation on a Release Candidate
+
+The bundle validation on a release candidate is specified in the Github Action job `validate-release-candidate-bundles`
+in `.github/workflows/bot.yml`. By default, this is disabled.
+
+To enable the bundle validation on a particular release candidate, makes the following changes to the job by fipping the
```

Comment:
we should update our release guide as well rgdn this step

Reply:
I'll put up a separate PR to update the release guide as a whole, as it's pretty out-dated and needs more work.